### ISSUE TITLE
Visitor can access private draft framework 

### ISSUE THREAD

**Issue by jmarks** [opensalt/opensalt#226]
If you visit a server as a visitor and get a link to a private draft framework. You can access the framework,

Similarly the CSE API is open for a private draft, and maybe should not be. Maybe we need a "Hidden draft" option for the case of shareable private frameworks, Open for discussion.

**Comment by roverwolf** [comment]
This was on purpose as per the original request when the (non-CASE specified status) *Private Draft* was added to OpenSALT.

The request was that the state would exist and not have it show up in the list on the home page or in the list of documents from the API, but that a link could be shared to allow others to view it (without having an account), especially since the intent behind CASE was that all CF Documents are public (otherwise they are not useful for exchange).

The one potential issue I see is that the URL is predictable, so that might be what needs to be resolved.

**Comment by brandonopened** [comment]
Yeah - the way we are going to handle it is the internal development instance will be behind a firewall. If it's considered that 'private' the org should probably just do the same as well.
